api/server/router/network/network_routes.go

```diff
@@ -112,15 +113,15 @@ func (n *networkRouter) getNetwork(ctx context.Context, w http.ResponseWriter, r
 
 	nw := n.backend.GetNetworks()
 	for _, network := range nw {
-		if network.ID() == term {
+		if network.ID() == term && (len(scope) == 0 || network.Info().Scope() == scope) {
```

Comment:
A `isMatchingScope(network, scope)` would be great here so it can be used in all 6 places.

not: I think comparing string to `""` makes the intent more obvious.